chia/rpc/wallet_rpc_client.py

```diff
@@ -240,18 +240,66 @@ async def select_coins(self, *, amount: int, wallet_id: int) -> List[Coin]:
         return [Coin.from_json_dict(coin) for coin in response["coins"]]
 
     # DID wallet
-    async def create_new_did_wallet(self, amount):
+    async def create_new_did_wallet(self, amount, fee=0, name="DID Wallet", backup_ids=[], required_num=0) -> Dict:
```

Comment:
use type annotations for the function arguments, for all the functions in this file

Reply:
Sounds like we need to remove this from the mypy exception list in main, address existing hinting issues, then that will be enforced here.